src/main/webapp/app/core/course/shared/course-sidebar/course-sidebar.component.html

```diff
@@ -116,7 +116,7 @@
                                 }
                             </div>
                         </a>
-                        @if (isLast && !isNavbarCollapsed()) {
+                        @if (isLast && !isNavbarCollapsed() && canExpand()) {
```

Comment:
Also here I cannot immediately grasp the condition in real words, a computed signal with a proper name would be a more clean way